File: JavaScript/01.JavaScript-Basic-February-2021/14.exams/2019.05.02_03-Exam/2.safari.js
function safari(input) {
    let budget = Number(input[0]);
    let oil = Number(input[1]);
    let day = input[2];

    let oilPrice = 2.10;
    let gidPrice = 100;

    let total = 0;
    let discount = 0;
    if (day == "Saturday") {
        discount = 0.1;
    } else if (day == "Sunday") {
        discount = 0.2;
    }

    total = (gidPrice + oil * oilPrice);
    total = total - total * discount;

    let diff = budget-total
    if (budget >= total) {
        console.log(`Safari time! Money left: ${diff.toFixed(2)} lv. `);
    } else {
        console.log(`Not enough money! Money needed: ${(Math.abs(diff).toFixed(2))} lv.`);
    }
}

safari(["1000", "10", "Sunday"]);
safari(["120", "30", "Saturday"]);
safari(["105.20", "15", "Sunday"]);

File: Python/01.Python-Basic-April-2019/15.exams/exam_2019.05.02/2.safari.py
budget = float(input())
fuel_lt = float(input())
week_day = input()

price_fuel = fuel_lt * 2.10

total = price_fuel + 100

if week_day == 'Saturday':
    total = total - 0.1 * total

elif week_day == 'Sunday':
    total = total - 0.2 * total

rest = abs(budget - total)

if budget >= total:
    print(f'Safari time! Money left: {rest:.2f} lv.')
else:
    print(f'Not enough money! Money needed: {rest:.2f} lv.')
